### PR TITLE
Devnet deployment

### PR DESCRIPTION
**Pull request by piobab**
devnet deployment set params from Osmosis mainnet red-bank contracts

### CODE REVIEW

File: scripts/deploy/osmosis/devnetConfig.ts [hunk 1]
```diff
@@ -0,0 +1,1018 @@
...
+    slope_2: '2',
+  },
+}
+
+export const ethAsset: AssetConfig = {
+  credit_manager: {
+    whitelisted: true,
+  },
+  symbol: 'ETH',
+  denom: eth,
```
Comment (markonmars) [inline scripts/deploy/osmosis/devnetConfig.ts:352]: Should this be wETH? Or is it native eth bridged via axelar?

**Review by markonmars — APPROVED** [review]

File: scripts/deploy/addresses/devnet.json
[the remark this answers is not in the document]
Comment (piobab) [reply scripts/deploy/addresses/devnet.json:1]: Yes we need credit-manager addr in address-provider. It is used by `params` contract for shared deeposit and in red-bank in few checks.

File: scripts/deploy/osmosis/devnetConfig.ts (hunk 1 above)
Comment (piobab) [reply scripts/deploy/osmosis/devnetConfig.ts:352]: It is wEth. This is just variable name but I will update it.

**Review by dancreee — APPROVED** [review]

File: scripts/deploy/osmosis/devnetConfig.ts [hunk 2]
```diff
@@ -0,0 +1,1018 @@
...
+  safetyFundFeeShare: '0.5',
+  deployerMnemonic: 'TODO',
+  slippage_tolerance: '0.01',
+  base_asset_symbol: 'OSMO',
+  // multisigAddr: 'osmo14w4x949nwcrqgfe53pxs3k7x53p0gvlrq34l5n',
+  runTests: false,
+  mainnet: false,
+  feeCollectorDenom: mars,
+  safetyFundDenom: axlUSDC,
+  swapRoutes: [
```
Comment (piobab) [inline scripts/deploy/osmosis/devnetConfig.ts:697]: @markonmars could you please take a look if we need more routes? For example for rewards distribution

Comment (markonmars) [reply scripts/deploy/osmosis/devnetConfig.ts:697]: Looks pretty comprehensive! We should still run a test though after migration to make sure

**Pull request merged**